wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Schema.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.schema
+
+expect class Schema internal constructor(protoFiles: Iterable<ProtoFile>) {
```

Comment:
Can we later put the whole schema into commonMain?

Reply:
That's the plan!